### PR TITLE
[STY][MAINT] Assorted file changes and added setup files

### PR DESCRIPTION
**Pull request by akimbler**
* Addresses #2

### CODE REVIEW

File: bidsutils/metadata.py [hunk 1]
```diff
@@ -61,8 +59,8 @@ def intended_for_gen(fmap_nifti, niftis):
 
 
 def complete_jsons(bids_dir, subs, ses, overwrite):
-    """
-    Assign 'IntendedFor' field to field maps in BIDS dataset.
+    """Assign 'IntendedFor' field to field maps in BIDS dataset.
+
```
Comment (tsalo) [inline bidsutils/metadata.py]: I'm going to push back on this pattern (both ways are PEP 257-compatible I believe) once we get around to numpydoc-ifying the docstrings, but for now it's fine.

**Comment by tsalo** [comment]
Do you know if this resolves the Docker issue revolving around how the package and repo are not named the same thing?

**Comment by akimbler** [comment]
> Do you know if this resolves the Docker issue revolving around how the package and repo are not named the same thing?

This does resolve it, I tested it by fresh installing on a `conda` env

File: bidsutils/metadata.py (hunk 1 above)
Comment (akimbler) [reply bidsutils/metadata.py]: I reverted them to a multi-line layout, but PEP 257 does require a summary line, which I added

Comment (tsalo) [reply bidsutils/metadata.py]: >Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line **may be on the same line as the opening quotes or on the next line.** The entire docstring is indented the same as the quotes at its first line (see example below).

Comment (tsalo) [reply bidsutils/metadata.py]: Ah, I see. You mean the separation of the one-line summary from the full description. Yes, that is good. I just mean the location of the one-line summary.

**Review by tsalo — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**